docs/data-sources/modelarts_dataset_versions.md

```diff
@@ -0,0 +1,68 @@
+---
+subcategory: "AI Development Platform (ModelArts)"
+---
+
+# huaweicloud_modelarts_dataset_versions
+
+Use this data source to get a list of ModelArts dataset versions.
+
+## Example Usage
+
+```hcl
+variable "dataset_id" {}
+data "huaweicloud_modelarts_dataset_versions" "test" {
+  dataset_id = var.dataset_id
+}
+```
+
+## Argument Reference
+
+The arguments of this data source act as filters for querying the available dataset versions in the current region.
+ All dataset versions that meet the filter criteria will be exported as attributes.
+
+* `region` - (Optional, String) Specifies the region in which to obtain dataset versions. If omitted, the
+provider-level region will be used.
```

Comment:
Please check the indentation format in the whole file

Reply:
done